Create new blogpost (#654)
open-life-science/open-life-science.github.io: _posts/2023-08-15-OLS-joining-forces-with-communities-to-broaden-participation-in-open-science.md

---
layout: post
title: OLS joining forces with communities to broaden participation in open science
authors:
- nyasita
- landimi2
- lauracion
- npalopoli
- karvovskaya
- malvikasharan
- estherplomp
- pazbc
- anelda
image: /images/2023-16-08-SouthAfrica-pic-OLS-8-blog-post.jpg
photos:
  name: Photo by Anelda Van der Walt
---

_OLS is happy to announce our collaboration with multiple international organisations and communities to deliver the Open Seeds OLS-8 programme_ 

> -----

Reviewer: ```suggestion
```

> Authors: Laurah Nyasita Ondari, Laura Acion, Nicolás Palopoli, Malvika Sharan, Andela Van de Valt, Lena Karvovskaya, Laura Ascenzi, Esther Plomp, Mike Landi Kofia, Paz Bernaldo

Author: ```suggestion
**Authors:** Laurah Nyasita Ondari, Laura Acion, Nicolás Palopoli, Malvika Sharan, Andela Van de Valt, Lena Karvovskaya, Laura Ascenzi, Esther Plomp, Mike Landi Kofia and Paz Bernaldo.
```

> **Authors:** Laurah Nyasita Ondari, Laura Acion, Nicolás Palopoli, Malvika Sharan, Andela Van de Valt, Lena Karvovskaya, Laura Ascenzi, Esther Plomp, Mike Landi Kofia and Paz Bernaldo.

Reviewer: Do we need that list if we have it in the top?

Reviewer 2: oh only added that because for example Laura Ascenzi isn't on github

Reviewer 2: but wait, she might be.... I'll double check

Reviewer: @NPDebs could you
-  add Laura to the people as if she is mot on GitHub
- add the corresponding id to the authors list in the metadata of the post
- remove the extra "authors" line

Thanks a lot

Author: Sure thing! 👍

Author: I've added [Laura](https://openlifesci.org/people#aliaslaurel). She was staring at us the whole time. 😄

Reviewer: Thanks a lot

> [OLS](https://openlifesci.org/) is joining forces with the [Bioinformatics Hub of Kenya initiative (BHKi)](https://bhki.org/),the South African Center for Digital Language Resources [(SADiLaR)](https://sadilar.org/index.php/en/) via the [ESCALATOR programme](https://escalator.sadilar.org/), [VU Amsterdam](https://vu.nl/en) and [MetaDocencia](https://www.metadocencia.org/), to bring together the newest [Open Seeds cohort](https://openlifesci.org/posts/2023/04/18/Introducing-the-new-OLS/), OLS-8. Our goal is to connect community members from our respective regions in the international open science network as well as strengthen their role as ambassadors for open science through local activities. 

Author: ```suggestion
[OLS](https://openlifesci.org/) is joining forces with the [Bioinformatics Hub of Kenya initiative (BHKi)](https://bhki.org/), the South African Center for Digital Language Resources [(SADiLaR)](https://sadilar.org/index.php/en/) via the [ESCALATOR programme](https://escalator.sadilar.org/), [VU Amsterdam](https://vu.nl/en) and [MetaDocencia](https://www.metadocencia.org/), to bring together the newest [Open Seeds cohort](https://openlifesci.org/posts/2023/04/18/Introducing-the-new-OLS/), OLS-8. Our goal is to connect community members from our respective regions in the international open science network, as well as strengthen their role as ambassadors for open science through local activities. 
```

> [OLS](https://openlifesci.org/) is joining forces with the [Bioinformatics Hub of Kenya initiative (BHKi)](https://bhki.org/), the South African Center for Digital Language Resources [(SADiLaR)](https://sadilar.org/index.php/en/) via the [ESCALATOR programme](https://escalator.sadilar.org/), [VU Amsterdam](https://vu.nl/en) and [MetaDocencia](https://www.metadocencia.org/), to bring together the newest [Open Seeds cohort](https://openlifesci.org/posts/2023/04/18/Introducing-the-new-OLS/), OLS-8. Our goal is to connect community members from our respective regions in the international open science network, as well as strengthen their role as ambassadors for open science through local activities. 

Reviewer: ```suggestion
[OLS]({{ site.url }}) is joining forces with the [Bioinformatics Hub of Kenya initiative (BHKi)](https://bhki.org/), the South African Center for Digital Language Resources [(SADiLaR)](https://sadilar.org/index.php/en/) via the [ESCALATOR programme](https://escalator.sadilar.org/), [VU Amsterdam](https://vu.nl/en) and [MetaDocencia](https://www.metadocencia.org/), to bring together the newest [Open Seeds cohort]({% link _posts/2023-04-18-Introducing-the-new-OLS.md %}), OLS-8. Our goal is to connect community members from our respective regions in the international open science network, as well as strengthen their role as ambassadors for open science through local activities. 
```

Leads and members from the partnering organisations and communities have worked with OLS since the very beginning. As OLS fellows, they have taken different roles in Open Seeds as well as provided the OLS team with several opportunities to learn from their work. 

> Who we are:

Author: ```suggestion
**Who we are:**
```

Open Seeds Programme Coordinator, Paz Bernaldo, the OLS Directors and team members will contribute to ensure that OLS-8 is delivered as planned. Likewise, OLS's existing collaborations, with TU Delft, [The Turing Way](https://the-turing-way.netlify.app/index.html), [ELIXIR](https://elixir-europe.org/) and [EOSC-Life](https://www.eosc-life.eu/), have been instrumental and will continue to be so in OLS-8. As an example, the [Faculty of Applied Sciences at TU Delft](https://www.tudelft.nl/en/faculty-of-applied-sciences/about-faculty), thanks to its Data Steward, Esther Plomp, has collaborated with Open Seeds since OLS-4 (2021), ensuring that participating PhD candidates [are eligible for 5 Discipline-related credits for their Doctoral Education Programme](https://estherplomp.github.io/TNW-OS-support/posts/open-science-OLS/).

Below we share the goals and visions behind these new partnerships and explain what they mean for the OLS-8 cohort.

> ## OLS-Africa

Reviewer: ```suggestion

## OLS-Africa

```

Reviewer 2: oh Yo had the idea of using Africa-OLS and MD-OLS instead of putting us first hehe

### BHKi

> _Introducing open science to learners early in their education, deepening their understanding and retaining them in science through international mentorship - Mike Landi and Laurah Nyasita, Co-leads of BHKi team_

Reviewer: ```suggestion

_Introducing open science to learners early in their education, deepening their understanding and retaining them in science through international mentorship - Mike Landi and Laurah Nyasita, Co-leads of BHKi team_
```

![Bioinformatics Hub of Kenya members, young students, approximately 6 women and five men, sitting in blue chairs forming a circle, inside a modern concrete building with high ceilings, many of them raising their hands as in voting, engaging in a conversation, the sunlight creates a square shape in the middle of the circle (BHKi)](/images/2023-16-08-BHki-pic-OLS-8-blog-post.jpg) BHKi members photographed during an outreach at Chuka University in Tharaka Nithi, Kenya.

### ESCALATOR by SADiLaR coordinated by Talarify

> _Bringing open science practices to the humanities and social sciences in South Africa through mentorship and embedment in a welcoming community of practice - Anelda Van der Walt, Director of Talarify_

Reviewer: ```suggestion

_Bringing open science practices to the humanities and social sciences in South Africa through mentorship and embedment in a welcoming community of practice - Anelda Van der Walt, Director of Talarify_
```

![A woman on a stage with banners and a podium giving a presentation to a group of people sitting at conference tables. A screen shows the home page of Voyant.](/images/2023-16-08-SouthAfrica-pic-OLS-8-blog-post.jpg) 
Mmasibidi Setaka, one of the OLS-8 participants, introducing people to Voyant, an open-source web-based application for text analysis, at DH-IGNITE KwaZulu Natal 2022.

## VU Amsterdam

> _Incentivising members in research infrastructure roles to become involved in the international open science movement - Lena Karvovskaya, Community Manager RDM and Open Science_ 

Reviewer: ```suggestion

_Incentivising members in research infrastructure roles to become involved in the international open science movement - Lena Karvovskaya, Community Manager RDM and Open Science_ 
```

We will support these projects in order to champion and exemplify open science in their departments and faculties. Through local events and newsletters, we will highlight their work and invite more members to engage with their work.

![Thirteen people posing for a photo, half of them sitting and hald standing, like in a primary school photo. The background is a modern building with dark greys and oranges, with copper walls. Their ages seem the go from mid twenties to their fifties](/images/2023-16-08-VU%20foto-MoniqueKooijmans-OLS-8-blog-post.jpg) Lena and the VU Amsterdam team

## MetaDocencia

> Building capacity to deliver long-term cohort-based training for Spanish-speaking researchers and technicians in Latin America - Laura Acion and Nicolás Palopoli, MetaDocencia Executive Co-Directors.

Author: ```suggestion
_Building capacity to deliver long-term cohort-based training for Spanish-speaking researchers and technicians in Latin America - Laura Acion and Nicolás Palopoli, MetaDocencia Executive Co-Directors._
```

> _Building capacity to deliver long-term cohort-based training for Spanish-speaking researchers and technicians in Latin America - Laura Acion and Nicolás Palopoli, MetaDocencia Executive Co-Directors._

Reviewer: ```suggestion

_Building capacity to deliver long-term cohort-based training for Spanish-speaking researchers and technicians in Latin America - Laura Acion and Nicolás Palopoli, MetaDocencia Executive Co-Directors._
```

![The image shows 12 people of different ages, informally dressed, in a natural garden, smiling and happy.](/images/2023-16-08-metadocencia-team-OLS-8-blog-post.jpg) Part of the MetaDocencia team and external collaborators at their first in-person retreat, held in Buenos Aires in 2023. 

## Making the most of this diverse cohort!

> Through this collaborative effort, we also want to reimagine some aspects of Open Seeds  so that better opportunities are made available for everyone. We would like all participants to  benefit from the diversity of expertise, perspectives and ideas members of this cohort and team will bring, and also engage with one another beyond their local contexts. In view os this goal we together thought of a couple of activities to implement in OLS-8:

Author: ```suggestion
Through this collaborative effort, we also want to reimagine some aspects of Open Seeds, so that better opportunities are made available for everyone. We would like all participants to  benefit from the diversity of expertise, perspectives and ideas members of this cohort and team will bring, and also engage with one another beyond their local contexts. In view of this goal, we together thought of a couple of activities to implement in OLS-8:
```

> Through this collaborative effort, we also want to reimagine some aspects of Open Seeds, so that better opportunities are made available for everyone. We would like all participants to  benefit from the diversity of expertise, perspectives and ideas members of this cohort and team will bring, and also engage with one another beyond their local contexts. In view of this goal, we together thought of a couple of activities to implement in OLS-8:

Reviewer: ```suggestion

Through this collaborative effort, we also want to reimagine some aspects of Open Seeds, so that better opportunities are made available for everyone. We would like all participants to  benefit from the diversity of expertise, perspectives and ideas members of this cohort and team will bring, and also engage with one another beyond their local contexts. In view of this goal, we together thought of a couple of activities to implement in OLS-8:
```

- We will host randomised peer-networking calls (3 mentees/project leads) each month. These calls will be a good place to meet informally, talk about their work (projects, struggles, successes, questions, etc.) or discuss assignments or presentations from the cohort calls. These calls will replace the OLS-7 Cafeterías that weren’t well attended. We hope these smaller self-organised meetings will work better for most people. 

Although not mandatory for graduating, we will encourage our cohort participants to engage in the community through various activities and channels. We believe this helps us all broaden our networks, increase our chances of collaboration and stay aware of the huge diversity of perspectives in science practise.   

# OLS-8 coming soon!

Because of our commitment to accessibility, we are also improving cohort activities considering feedback given by past cohort members. We will share more details in a future post.

We are excited to welcome our new cohort members together with our partnering organisations and kick off the Open Seeds OLS-8 in September. 

> Do you have an idea or feedback that you'd like to share with us? Please email team@openlifesci.org, or connect with Paz on Slack if you are an OLS member.

Reviewer: ```suggestion
Do you have an idea or feedback that you'd like to share with us? Please email [{{ site.email }}](mailto:{{ site.email }}), or connect with Paz on Slack if you are an OLS member.
```

> Funding acknowledgement: This cohort is funded by the Wellcome Trust Open Research Fund and the CZI grant awarded to OLS in 2022.

Author: ```suggestion
**Funding acknowledgement:** This cohort is funded by the Wellcome Trust Open Research Fund and the CZI grant awarded to OLS in 2022.
```

---
layout: post
title: OLS joining forces with communities to broaden participation in open science
authors:
- nyasita
- landimi2
- lauracion
- npalopoli
- karvovskaya
- malvikasharan
- estherplomp
- pazbc
- anelda

> image: https://images/2023-16-08-SouthAfrica-pic-OLS-8-blog-post.jpg

Reviewer: ```suggestion
image: /images/2023-16-08-SouthAfrica-pic-OLS-8-blog-post.jpg
```

Open Seeds Programme Coordinator, Paz Bernaldo, the OLS Directors and team members will contribute to ensure that OLS-8 is delivered as planned. Likewise, OLS's existing collaborations, with TU Delft, [The Turing Way](https://the-turing-way.netlify.app/index.html), [ELIXIR](https://elixir-europe.org/) and [EOSC-Life](https://www.eosc-life.eu/), have been instrumental and will continue to be so in OLS-8. As an example, the [Faculty of Applied Sciences at TU Delft](https://www.tudelft.nl/en/faculty-of-applied-sciences/about-faculty), thanks to its Data Steward, Esther Plomp, has collaborated with Open Seeds since OLS-4 (2021), ensuring that participating PhD candidates [are eligible for 5 Discipline-related credits for their Doctoral Education Programme](https://estherplomp.github.io/TNW-OS-support/posts/open-science-OLS/).

**Below we share the goals and visions behind these new partnerships and explain what they mean for the OLS-8 cohort.**

> ## Africa-OLS

Reviewer: ```suggestion

## Africa-OLS

```

![Bioinformatics Hub of Kenya members, young students, approximately 6 women and five men, sitting in blue chairs forming a circle, inside a modern concrete building with high ceilings, many of them raising their hands as in voting, engaging in a conversation, the sunlight creates a square shape in the middle of the circle (BHKi)](/images/2023-16-08-BHki-pic-OLS-8-blog-post.jpg) BHKi members photographed during an outreach at Chuka University in Tharaka Nithi, Kenya.

### ESCALATOR by SADiLaR coordinated by Talarify

_Bringing open science practices to the humanities and social sciences in South Africa through mentorship and embedment in a welcoming community of practice - Anelda Van der Walt, Director of Talarify_

> **ESCALATOR aims to support the development of an inclusive and active community of practice in Digital Humanities and Computational Social Science in South Africa**. The programme is hosted by [SADiLaR](https://sadilar.org/index.php/en/), one of the South African Research Infrastructure Roadmap entities that receives its funding from the Department of Science and Innovation. ESCALATOR is an ambitious initiative built on three pillars: [DH-IGNITE](https://dh-ignite.org), an in-person or hybrid two-day community event run in five different regions across the country; the [Digital Champions Initiative] (https://escalator.sadilar.org/champions/overview/),  for people with different levels of digital and computational experience; and the [Stakeholder Map](https://escalator.sadilar.org/stakeholder-map/), a database and interactive visualisation of people and resources. In 2021, before kicking off the Digital Champions Initiative, the ESCALATOR team partnered with a number of leaders from existing mentorship programmes to run a [Mentorship Indaba](https://escalator.sadilar.org/post/2021/05/2021-05-03-mentorship-indaba/) with the aim of building on the lessons learnt through the development and implementation of these programmes. One of these programmes was OLS. This resulted in an article titled [“Ten simple rules for establishing a mentorship programme”](https://doi.org/10.1371/journal.pcbi.1010015). 

Reviewer: ```suggestion
**ESCALATOR aims to support the development of an inclusive and active community of practice in Digital Humanities and Computational Social Science in South Africa**. The programme is hosted by [SADiLaR](https://sadilar.org/index.php/en/), one of the South African Research Infrastructure Roadmap entities that receives its funding from the Department of Science and Innovation. ESCALATOR is an ambitious initiative built on three pillars: 
- [DH-IGNITE](https://dh-ignite.org), an in-person or hybrid two-day community event run in five different regions across the country; 
- the [Digital Champions Initiative] (https://escalator.sadilar.org/champions/overview/),  for people with different levels of digital and computational experience; 
- the [Stakeholder Map](https://escalator.sadilar.org/stakeholder-map/), a database and interactive visualisation of people and resources. 

In 2021, before kicking off the Digital Champions Initiative, the ESCALATOR team partnered with a number of leaders from existing mentorship programmes to run a [Mentorship Indaba](https://escalator.sadilar.org/post/2021/05/2021-05-03-mentorship-indaba/) with the aim of building on the lessons learnt through the development and implementation of these programmes. One of these programmes was OLS. This resulted in an article titled ["Ten simple rules for establishing a mentorship programme"](https://doi.org/10.1371/journal.pcbi.1010015). 
```

**We will support these projects in order to champion and exemplify open science in their departments and faculties**. Through local events and newsletters, we will highlight their work and invite more members to engage with their work.

![Thirteen people posing for a photo, half of them sitting and hald standing, like in a primary school photo. The background is a modern building with dark greys and oranges, with copper walls. Their ages seem the go from mid twenties to their fifties](/images/2023-16-08-VU%20foto-MoniqueKooijmans-OLS-8-blog-post.jpg) Lena and the VU Amsterdam team

## MetaDocencia

_Building capacity to deliver long-term cohort-based training for Spanish-speaking researchers and technicians in Latin America - Laura Acion and Nicolás Palopoli, MetaDocencia Executive Co-Directors._

> MetaDocencia launches its work in [two of the three NASA Transform to Open Science (TOPS)](https://www.metadocencia.org/en/post/nasatops/) mission grants by joining OLS-8 at all levels. That is, **the whole MetaDocencia team, depending on where each person is, will be participating in OLS-8 as mentees, facilitators, and mentors. This participation will allow MetaDocencia and OLS teams to start collaborating in view of the two NASA projects we share.** MetaDocencia will also learn OLS’ business model from the inside so we can contextualise it to our region. At the same time, participating in OLS-8 will strengthen South-South collaborations in a way our community has yet to experience. Throughout OLS-8, MetaDocencia will be working in two mentored groups. One will lead MetaDocencia’s yearly governance and policies review. The other one will build the structure and infrastructure for offering virtual cohorts to Spanish-speaking researchers and technicians in Latin America. These learnings will be shared with the broader community openly, allowing others to also learn from our experience.

Reviewer: ```suggestion
MetaDocencia launches its work in [two of the three NASA Transform to Open Science (TOPS)](https://www.metadocencia.org/en/post/nasatops/) mission grants by joining OLS-8 at all levels. That is, **the whole MetaDocencia team, depending on where each person is, will be participating in OLS-8 as mentees, facilitators, and mentors. This participation will allow MetaDocencia and OLS teams to start collaborating in view of the two NASA projects we share.** MetaDocencia will also learn OLS' business model from the inside so we can contextualise it to our region. At the same time, participating in OLS-8 will strengthen South-South collaborations in a way our community has yet to experience. Throughout OLS-8, MetaDocencia will be working in two mentored groups. One will lead MetaDocencia's yearly governance and policies review. The other one will build the structure and infrastructure for offering virtual cohorts to Spanish-speaking researchers and technicians in Latin America. These learnings will be shared with the broader community openly, allowing others to also learn from our experience.
```

![The image shows 12 people of different ages, informally dressed, in a natural garden, smiling and happy.](/images/2023-16-08-metadocencia-team-OLS-8-blog-post.jpg) Part of the MetaDocencia team and external collaborators at their first in-person retreat, held in Buenos Aires in 2023. 

## Making the most of this diverse cohort!

> **Through this collaborative effort, we also want to reimagine some aspects of Open Seeds**, so that better opportunities are made available for everyone. We would like all participants to benefit from the diversity of expertise, perspectives and ideas members of this cohort and team will bring, and also engage with one another beyond their local contexts. In view of this goal, we together thought of a couple of activities to implement in OLS-8:

Reviewer: ```suggestion

**Through this collaborative effort, we also want to reimagine some aspects of Open Seeds**, so that better opportunities are made available for everyone. We would like all participants to benefit from the diversity of expertise, perspectives and ideas members of this cohort and team will bring, and also engage with one another beyond their local contexts. In view of this goal, we together thought of a couple of activities to implement in OLS-8:
```

- We will host **randomised peer-networking calls** (3 mentees/project leads) each month. These calls will be a good place to meet informally, talk about their work (projects, struggles, successes, questions, etc.) or discuss assignments or presentations from the cohort calls. These calls will replace the OLS-7 Cafeterías that weren’t well attended. We hope these smaller self-organised meetings will work better for most people. 

Although not mandatory for graduating, we will encourage our cohort participants to engage in the community through various activities and channels. We believe this helps us all broaden our networks, increase our chances of collaboration and stay aware of the huge diversity of perspectives in science practise.   

# OLS-8 coming soon!

We are excited to welcome our new cohort members together with our partnering organisations and kick off the Open Seeds OLS-8 in September. 

> Do you have feedback for us? Please email team@we-are-ols.org, or connect with Paz on Slack if you are an OLS member.

Reviewer: ```suggestion
Do you have feedback for us? Please email [{{ site.email }}](mailto:{{ site.email }}), or connect with Paz on Slack if you are an OLS member.
```
